packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -915,6 +921,20 @@ export function TransferPanelMain({
                     forToken={selectedToken}
                     prefix={selectedToken ? 'BALANCE: ' : ''}
                   />
+                  {app.isDepositMode &&
+                    isTokenMainnetUSDC(selectedToken?.address) && (
+                      <TokenBalance
+                        balance={
+                          erc20L2Balances?.[CommonAddress.ArbitrumOne.USDC] ??
+                          constants.Zero
+                        }
+                        on={NetworkType.l2}
+                        forToken={Object.assign({}, selectedToken, {
+                          symbol: 'USDC'
+                        })}
+                        skipUSDCoverride
```

Comment:
We don't really need to sanitize anything, we could just hardcode `USDC` as it's a special case. That will also get rid of the change that includes the `skipUSDCoverride`, which I think are unnecessary.